apis/openai_resource.py

```
@@ -0,0 +1,67 @@
+api = Namespace('openai', description=msg.API_NAMESPACE_OPENAI_DESCRIPTION)
+
+ALLOWED_EXTENSIONS = {'wav', 'm4a', 'mp3', 'mp4'}
+def get_transcription(path):
+  try:
+      audio_file = open(path, 'rb')
+      response = openai.Audio.transcribe(
+        model="whisper-1",
+        file=audio_file
+      )
+      print(response)
+      return response.text 
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)